From: Romain Janvier <romain.janvier@hotmail.fr>
Date: Sat, 12 Jan 2013 16:50:05 +0100
Subject: added distutils stuff (setup.py)

---
 setup.py |   17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 setup.py

diff --git a/setup.py b/setup.py
new file mode 100644
index 0000000..068536b
--- /dev/null
+++ b/setup.py
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='img2dcm',
+	version='0.1.0',
+	description='Convert bmp, tif, png, jpg, vti to dcm',
+	author='Thiago Franco de Moraes',
+	author_email='invesalius@cti.gov.br',
+	url='https://bitbucket.org/tfmoraes/img2dcm',
+	maintainer="Romain Janvier",
+	maintainer_email="romain.janvier@hotmail.fr",
+	py_modules=['ivDicom'],
+	scripts=['img2dcm'],
+	licence='GPL2',
+	requires=['vtk (>=0.4.67)', 'gdcm (>=2.2.0)', 'vtkgdcm (>=2.2.0)'],
+     )
